flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/logical/FlinkLogicalWindowAggregate.scala

```diff
@@ -103,6 +106,19 @@ class FlinkLogicalWindowAggregateConverter
     FlinkConventions.LOGICAL,
     "FlinkLogicalWindowAggregateConverter") {
 
+  override def matches(call: RelOptRuleCall): Boolean = {
+    val agg = call.rel(0).asInstanceOf[LogicalWindowAggregate]
+
+    // we do not support these functions natively
+    // they have to be converted using the WindowAggregateReduceFunctionsRule
+    val supported = agg.getAggCallList.asScala.map(_.getAggregation.getKind).forall {
+      case SqlKind.STDDEV_POP | SqlKind.STDDEV_SAMP | SqlKind.VAR_POP | SqlKind.VAR_SAMP => false
```

Comment:
How about AVG and SUM? they are also in AggregateReduceFunctionsRule. Also, I think it's better to use SqlKind.AVG_AGG_FUNCTIONS.contains() or AggregateReduceFunctionsRule.isReducible() (it's private now though) here in the case statement. So it will keep consistent if calcite changes.

Reply:
We have a built-in function for AVG (which we don't really need anymore) and SUM, so we could translate such plans.
But I agree, using SqlKind.AVG_AGG_FUNCTIONS.contains() is better.

Reply:
Replacing the current code by `SqlKind.AVG_AGG_FUNCTIONS.contains()` lead to several test failures. These tests expected an `AVG` aggregation function that was now replaced by `SUM / COUNT`.

Reply:
How about SqlKink.AVG_AGG_FUNCTIONS.contains(kind) && kind != SqlKind.SUM && kind != SqlKind.AVG?

Reply:
sounds good to me. Will update the PR